packages/vscode-extension/package.json

```diff
@@ -424,6 +425,7 @@
     "bundle-webview-controllers": "esbuild ./src/webviews/controllers/* --bundle --outdir=dist/webviews/ --platform=browser",
     "build": "tsc -b && pnpm gen-textmate && pnpm bundle-extension && pnpm bundle-language-server && pnpm bundle-webview-controllers",
     "build:dev": "tsc -b && pnpm gen-textmate && pnpm bundle-extension:dev && pnpm bundle-language-server && pnpm bundle-webview-controllers",
+    "package": "pnpm vsce package --pre-release --no-dependencies",
```

Comment:
The release job to VSCode was broken because the tool to generate the VSCode packages does not work with pnpm: https://github.com/microsoft/vscode-vsce/issues/421.

Luckily we are bundling the plugin so it works if we pass the flag `--no-dependencies` but this is one to keep track of.

Reply:
Oh that's unfortunate, thanks for looking into it